Week-18/Todo-list-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "todo-list-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.14.9",
+    "@mui/material": "^5.14.9",
+    "@reduxjs/toolkit": "^1.9.5",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.1.2"
```

Comment:
Are you using this package? I saw usage for reduxjs-toolkit only. You can remove unused packages.

Reply:
can i remove react-dom, i will not create any issue?

Reply:
No, I’m talking about react-redux